src/routes/(app)/[username]/+page.svelte

```diff
@@ -276,18 +277,22 @@
 				</div>
 			{/if}
 			{#if data.pages.length > 0}
-				<div>
-					<h2 class="mb-4 text-center font-rubik text-2xl font-bold">Pages</h2>
-					<ul class="mt-6 flex flex-col items-center gap-10">
-						{#each data.pages as p}
-							<li>
-								<a class="link" href={`/${$page.params.username}/${p.slug}`}>
-									{p.name || p.slug}
-								</a>
-							</li>
-						{/each}
-					</ul>
-				</div>
+				{#if editingState.editing&&import.meta.env.PUBLIC_ENABLE_EXPERIMENTS}
```

Comment:
The environment variable should actually be accessed through the `$env/dynamic/public` import so that it can be configured at runtime when deployed like we do here:

https://github.com/muni-town/weird/blob/118dd2092bebcb3832cfb09bb391f60a143801a2/src/routes/(app)/account/github/%2Bpage.svelte#L2

https://github.com/muni-town/weird/blob/118dd2092bebcb3832cfb09bb391f60a143801a2/src/routes/(app)/account/github/%2Bpage.svelte#L5